src/jit/codegencommon.cpp

```diff
@@ -4608,13 +4620,21 @@ void CodeGen::genCheckUseBlockInit()
     // zero init the stack using a block operation instead of a 'case by case' basis.
     genInitStkLclCnt = initStkLclCnt;
 
-    /* If we have more than 4 untracked locals, use block initialization */
-    /* TODO-Review: If we have large structs, bias toward not using block initialization since
-       we waste all the other slots.  Really need to compute the correct
-       and compare that against zeroing the slots individually */
+/* If we have more than 4 untracked locals, use block initialization */
+/* TODO-Review: If we have large structs, bias toward not using block initialization since
+   we waste all the other slots.  Really need to compute the correct
+   and compare that against zeroing the slots individually */
```

Comment:
I do not quite understand this comment... Could you please improve it with more details if it is still useful?

Reply:
I agree. I don't understand why slots need to be zeroed individually. This should just be linear stack space and it is all being zeroed, so you should be able to just zero-it however is most efficient for that size.

Reply:
Not sure I completely understand it either -- I think the comment is saying we should do a more thorough examination of GC structs and bias towards field by field zeroing when there are large GC structs with relatively few GC fields (at least, when we're not also zeroing the non-gc fields).